src/lxc/cmd/lxc_usernsexec.c

```diff
@@ -399,9 +401,13 @@ int main(int argc, char *argv[])
 
 	close(pipe_fds1[1]);
 	close(pipe_fds2[0]);
-	if (lxc_read_nointr(pipe_fds1[0], buf, 1) < 1) {
+	ret = lxc_read_nointr(pipe_fds1[0], buf, 1);
+	if (ret < 0) {
 		perror("read pipe");
 		exit(EXIT_FAILURE);
+	} else if (ret == 0) {
+		fprintf(stderr, "read pipe: EOF\n");
```

Comment:
Could you please make this `Failed to read from pipe\n"`?

Reply:
Done